packages/node_modules/@webex/react-component-utils/src/adaptiveCardUtil.js

```diff
@@ -100,11 +103,17 @@ export function getAdaptiveCard(
     const adaptiveCard = new PatchedAdaptiveCard.AdaptiveCard();
 
     adaptiveCard.onExecuteAction = (action) => {
+      const btnClicked = action.renderedElement;
```

Comment:
Since this constant is only used in the `ADAPTIVE_CARD_SUBMIT_ACTION` case, let's move the definition into there.

Reply:
@adamweeks  moved to local scope